### ISSUE TITLE
iOS - Issue after getting : Error 17

### ISSUE THREAD

**Issue by HamzaLJ** [arielhernandezmusa/capacitor-biometric-auth#2]
In case the user's face ID didn't work, the user is asked to enter the passcode and by that is getting Error **17** however, the OS passcode screen is never shown.

Currently on iOS: 13

```
Ionic:

   Ionic CLI                     : 5.4.13 
   Ionic Framework               : @ionic/angular 5.0.0-dev.202001032123.aac4bb0
   @angular-devkit/build-angular : 0.803.21
   @angular-devkit/schematics    : 8.3.21
   @angular/cli                  : 8.3.21
   @ionic/angular-toolkit        : 2.1.1

Capacitor:

   Capacitor CLI   : 1.4.0
   @capacitor/core : 1.4.0

Cordova:

   Cordova CLI       : 7.1.0
   Cordova Platforms : none
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.0, (and 10 other plugins)

Utility:

   cordova-res : 0.8.1
   native-run  : 0.3.0

System:

   ios-deploy : 1.9.2
   ios-sim    : 6.0.0
   NodeJS     : v10.10.0 (/usr/local/bin/node)
   npm        : 6.11.2
   OS         : macOS Mojave
   Xcode      : Xcode 11.2.1 Build version 11B500
```

**Comment by arielhernandezmusa** [comment]
@HamzaLJ , did you update your Info.plist with Privacy - Face ID Usage Description?

**Comment by leo-freitas** [comment]
I'm hit with the same bug. My Info.plist has the usage description correctly set (`<key>NSFaceIDUsageDescription</key>`)

**Comment by arielhernandezmusa** [comment]
Hi, I test and is working, see an example here

https://github.com/arielhernandezmusa/capacitor-camera-preview-demos

**Comment by arielhernandezmusa** [comment]
@HamzaLJ , I just fix this error and publish the version 0.0.8
